### DISCUSSION TITLE
But.. How?

### DISCUSSION THREAD

**Discussion by UkrainianBanderasCat** [UkrainianBanderasCat/Digital-Logic-Sim2#40] [General]
Hey, but how does chip importing work? When I'm trying to import a chip it just says error...

**Comment by UkrainianBanderasCat** [comment]
O.. I found error. Assets/Scripts/Save System/ChipImporter.cs(26,23): error CS0103: The name 'EditorUtility' does not exist in the current context. We can't build it :(, but we need to build it

**Reply by firecerne** [reply]
I'll try to fix that, but it will be more complicated.

**Reply by firecerne** [reply]
Would you mind if I use an asset from the Unity store ? It is free, doesn't seem out of place in the game, and is very simple.

**Reply by UkrainianBanderasCat** [reply]
Sure, you can use it
